app/src/main/java/com/mensinator/app/PeriodDatabaseHelper.kt

```diff
@@ -401,6 +404,15 @@ class PeriodDatabaseHelper(context: Context) :
         return setting
     }
 
+    override fun getStringSettingByKey(key: String): String {
```

Comment:
Small improvement to log & code struture (avoiding `var`)
```
    override fun getStringSettingByKey(key: String): String {
        val string = getSettingByKey(key)?.value
        if (string == null) {
            Log.e("getStringSettingByKey", "key '$key' is null")
            return "Unknown"
        }
        return string
    }
```

Reply:
nice, changed